alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -51,3 +51,12 @@ test-rpm:
 	[[ ! -d ./_output/ ]]
 	$(MAKE) clean
 .PHONY: test-rpm
+
+test-codegen:
+	$(MAKE) update-codegen
+	$(MAKE) verify-codegen
+	./_output/tools/bin/yq m -i manifests/operator.openshift.io_myoperatorresources.crd.yaml manifest.test.crd.yaml-merge-patch
```

Comment:
I suppose `echo aaa >> the_file && echo bbb >> a_new_file` would be enough to test it is not matching for the diff

Reply:
you should not get any error on garbage if you change the generated files, if that's the case the generator has loop (it shouldn't depend on it's output)